### DISCUSSION TITLE
Is possible to look inside of model when the camera zoom in?

### DISCUSSION THREAD

**Discussion by 186dk** [google/model-viewer#1899] [Q&A]

**Comment by elalish** [comment]
Actually we worked hard to ensure that would not happen, but if you set the radius of `minCameraOrbit` to less than what we calculate for `auto`, then this becomes possible.

**Comment by 186dk** [comment]
I tried to set minCameraOrbit="-1000", but it didn't work. I have a 3d house model, the model-viewer need to able to show both outside and inside of the house model. How can I achieve that? Thanks.

**Comment by elalish** [comment]
@186dk If you look at the format of `minCameraOrbit` on [modelviewer.dev](https://modelviewer.dev/), you'll see that the format is `"theta phi radius"` and that each needs units. You could try `"auto auto 0m"`.

**Reply by ikarus-pritish** [reply]
This was helpful. Thanks @elalish
